lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -2547,12 +2547,24 @@ def _add_image_features(self, node_space: schema.NodeSpace) -> None:
         else:
             ...
 
-    def _load_image_features(self, node_space: schema.NodeSpace) -> None:
+    def _check_image_capability(self, node_space: schema.NodeSpace) -> None:
+        azure_runbook = node_space.get_extended_runbook(AzureNodeSchema, AZURE)
+        if azure_runbook.vhd:
+            if node_space.network_interface:
+                data_path = search_space.intersect_setspace_by_priority(  # type: ignore
+                    node_space.network_interface.data_path,
+                    azure_runbook.vhd.network_data_path,
+                    [],
```

Comment:
Use the priority in schema.py

Reply:
In the function intersect_setspace_by_priority, the parameter priority_list isn't used. In this function, it has the below comment:
# intersect doesn't need to take care about priority.